.gitignore

```diff
@@ -6,6 +6,7 @@ TestResults/
 _ReSharper.*/
 packages/
 artifacts/
+fakeinstall/
```

Comment:
We can designate the `%USERPROFILE%\.omnisharp\local` / `$HOME/.omnisharp/local` as the output by default. And we can also allow user to override the path in `build.json`. Try not to have default value hardcoded.